test/functional/legacy/089_number_relativenumber_spec.lua

```diff
@@ -0,0 +1,86 @@
+-- - Some tests for setting 'number' and 'relativenumber'
+--   This is not all that useful now that the options are no longer reset when
+--   setting the other.
+
+local helpers = require('test.functional.helpers')
+local feed = helpers.feed
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('setting \'number\' and \'relativenumber\'', function()
+  setup(clear)
+
+  it('is working', function()
+    execute('set hidden nu rnu')
```

Comment:
I think these `execute()` blocks can be one big `source([[...]])` call. Example: https://github.com/neovim/neovim/pull/2029/files#diff-f91bb1d2e99bfd9a4eac66290bdf501fR18